### ISSUE TITLE
Duplicate `deploy` section on docker swarm docker-compose file

### ISSUE THREAD

**Issue by stratosgear** [zatosource/zato-build#70]
There is a duplicate `deploy` section on https://github.com/zatosource/zato-build/blob/master/docker/swarm/docker-compose.yml.template#L105 (there is another one on line 127)

This causes issues when running the `dockerize` command.

**Comment by dsuch** [comment]
Thank you for the ticket, we will have a look soon.

**Comment by stratosgear** [comment]
Sorry, I spoke too fast.

I followed the while installation workflow from https://zato.io/docs/admin/guide/install/py3/swarm.html on a newly created cloud instance and it got working without issues from dockerize (basically dockerize does not care what it templates)

What I meant to say in my first attempt, is that the generated `docker-compose.yml` is not valid, because it contains duplicate keys.  VScode hilighted the duplicate key and I noticed it.

Sorry for not being clear the first time around.  I think the issue still stands, though!
